src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,46 @@
+.forecast-details {
+  background-color: white;
+  color: black;
+  width: fit-content;
+  margin: auto;
+  padding: 1rem;
+  border-radius: 5px;
```

Comment:
It may be worth adding `text-align: right;` to this declaration as most of the child elements of the element with this style applied would inherit it.